nxt/sensor/hitechnic.py

```diff
@@ -459,9 +460,238 @@ def set_digital_modes(self, modes):
         """Sets input/output mode of digital pins. Can take a Digital_Data() object."""
         self.write_value('digital_cont', (int(modes), ))
 
+
 Prototype.add_compatible_sensor(None, 'HiTechnc', 'Proto   ')
 
 
+class SuperPro(BaseDigitalSensor):
+    """
+    Object for HiTechnic sensor SuperPro boards.
+    """
+    I2C_ADDRESS = BaseDigitalSensor.I2C_ADDRESS.copy()
+    I2C_ADDRESS.update({
+        "version": (0x00, "8s"),
+        "manufacturer": (0x08, "8s"),
+        "sensor_type": (0x10, "8s"),
+        "analog_a0": (0x42, "<H"),
+        "analog_a1": (0x44, "<H"),
+        "analog_a2": (0x46, "<H"),
+        "analog_a3": (0x48, "<H"),
+        "digital_in": (0x4C, "<B"),
+        "digital_out": (0x4D, "<B"),
+        "digital_dir": (0x4E, "<B"),
+        "strobe_out": (0x50, "B"),
+        "led_out": (0x51, "B"),
+        "analog_out0_mode": (0x52, "B"),
+        "analog_out0_freq": (0x53, "<H"),
+        "analog_out0_volts": (0x55, "H"),
+        "analog_out1_mode": (0x57, "B"),
+        "analog_out1_freq": (0x58, "<H"),
+        "analog_out1_volts": (0x5A, "H")
+    })
+
+    """
+    Some users online report that this I2C device address is 0x08, the documentation states that it's 0x10.
+    Let us know if this value doesn't work for you and 0x08 does. 0x10 works in testing with my hardware.
```

Comment:
LSB would mean `0x09` or `0x11` though, right? I think the comment still has a chance of being useful; there's no way of being certain that it couldn't be because we didn't develop the firmware on these systems and there could be different revisions or something, particularly if the SuperPro (developed later) has a different address from the Prototype board.

Alternatively, I could try reaching out in those online forums or reading into it more perhaps and ask/conduct some research.